CHANGELOG/CHANGELOG-1.15.md

```diff
@@ -0,0 +1,86 @@
+
+- The built-in protocols incude Modbus, Opc-UA and Bluetooth are removed in device instance, and the built-in mappers for these proytocols
+still exists and will be maintained and updated to latest verison.
+- Users must define the protocol config through `CustomizedValue` in `ProtocolConfig`.
+- DMI date plane related fields are added, users can config the collection and reporting frequency of device data, and the destination
+to whcih(such as database, httpserver) data is pushed.
+- Controls whether to report device data to cloud.
+
+Refer to the link for more details. ([#4983](https://github.com/kubeedge/kubeedge/pull/4983))
+
+
+### Support Alpha version of DMI DatePlane and Mapper-Framework
+
+Alpha version of DMI date plane is supported, DMI date plane is mainly implemented in mapper, providing interface for
+pushing data, pulling data, and storing data in database.
+
+To make writing mapper easier, a mapper development framework subproject **Mapper-Framework** is provided in this release.
+Mapper-Framework provides mapper runtime libs and tools for scaffolding and code generation to bootstrap a new mapper project.
+Users only need to run a command `make generate` to generate a mapper project, then add protocol related code to mapper.
+
+Refer to the link for more details. ([#5023](https://github.com/kubeedge/kubeedge/pull/5023))
+
+
+### Support Kubernetes native Static Pod on Edge Nodes
+Kubernetes native `Static Pod` is supported on edge node in this release. Users can create pods on edge nodes by place pod manifests in
+`/etc/kubeedge/manifests`, same as that on the Kubernetes node.
+
+Refer to the link for more details. ([#4825](https://github.com/kubeedge/kubeedge/pull/4825))
+
+
+### Support more Kubernetes Native Plugin Running on Edge Node
+
+Kubernetes non-resource kind request `/version` is supported from edge node, users now can do `/version` requests in edge node from metaserver.
+In addition, it can easily support other non-resource kind of requests like `/healthz` in edge node with the curent framework.
+Many kubernetes plugins like cilium/calico which depend on these non-resource kind of requests, now can run on edge nodes.
+
+Refer to the link for more details. ([#4904](https://github.com/kubeedge/kubeedge/pull/4904))
+
+### Upgrade Kubernetes Dependency to v1.26.7
+
+Upgrade the vendered kubernetes version to v1.26.7, users are now able to use the feature of new version on the cloud and on the edge side.
+
+Refer to the link for more details. ([#4929](https://github.com/kubeedge/kubeedge/pull/4929))
+
+
+## Important Steps before Upgrading
+
+- In KubeEdge v1.15, new v1beta1 version of device API is incompatible with earlier versions of v1alpha1, users need to update the device API yamls to v1bata1 if you want to use v1.15.
+- In KubeEdge v1.14, EdgeCore has removed the dockeshim support, so users can only use `remote` type runtime, and uses `containerd` runtime by default. If
```

Comment:
One more step need to be added:
In KubeEdge v1.15,  you will need to upgrade to containerd version 1.6.0 or later. Containerd minor version 1.5 and older will not be supported in KubeEdge v1.15.

Reply:
https://kubernetes.io/blog/2022/11/18/upcoming-changes-in-kubernetes-1-26/#cri-api-removal

Reply:
Done.